Apps/AdminWebClient/src/ClientApp/src/utility/masks.ts

```diff
@@ -8,7 +8,24 @@ export function zipCodeMask(value: string): MaskCharacter[] {
     return numbers.length > 5 ? extendedFormat : zipFormat;
 }
 
+export function phoneNumberMaskTemplate(): MaskCharacter[] {
```

Comment:
Why is this mask so much more complicated than the ones immediately below?

Reply:
@sslaws we have a bug for the mask Bug #12467

Reply:
We were experimenting with the mask format because applying the mask was preventing the component from loading.